### PR TITLE
Update to v3

### PR DESCRIPTION
**Pull request by mikemand**
I used `master` as the base branch and then merged the `2.0` branch into mine. Even though I told Git to add a merge commit, it didn't (ugh, Git -.-).

I probably didn't even need to move the sidebar; I could have removed it instead... Oh well, I'll let you decide that. :D

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,5 @@
-.idea/
```
Comment (JulienTant) [inline .gitignore]: Why did you remove this one ?

Comment (mikemand) [reply .gitignore]: I copied the `.gitignore` from the Blog module.

It shouldn't even be there, in the first place though... Stuff like that should be in the individual developer's global `.gitignore`. See [here](https://thomashunter.name/blog/global-gitignore-vs-repository-gitignore/) for more information.

Comment (JulienTant) [reply .gitignore]: > It shouldn't even be there, in the first place though... Stuff like that should be in the individual developer's global .gitignore. See here for more information.

I can't agree more, but it's quite difficult in the case of an open source project to force user do that. It's a security.

That being said, since v2 and the module subsplitting we've been building with Nicolas, nobody will open just the blog module and commit here so... yeah my comment useless i guess :)

Comment (mikemand) [reply .gitignore]: Technically Nick can force the contributor to do it himself. :D Just don't merge the PR until they do.

My only objection to adding it is that once you add one editor's metafiles to be ignored, it opens the gates for everyone else to want to add their editor's as well.

Comment (nWidart) [reply .gitignore]: I agree to remove it from the gitingore. No other repo has it, this is probably something from very early on.

**Comment by nWidart** [comment]
As a reminder, pusher config:

``` php
'pusher' => [
    'driver' => 'pusher',
    'key' => env('PUSHER_APP_KEY'),
    'secret' => env('PUSHER_APP_SECRET'),
    'app_id' => env('PUSHER_APP_ID'),
    'options' => [
        'cluster' => 'us2',
        'encrypted' => true,
    ],
],
```

**Comment by nWidart** [comment]
Can you add:

```json
"suggest": {
    "pusher/pusher-php-server": "Allows notifications to enable real-time notifications."
  },
```
